From daec8ce5fbd946e8836a13d461f0be805de8b3fd Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Sat, 10 Dec 2011 01:14:58 +0800
Subject: [PATCH 179/259] ENGR00169783 vpu: Fix cropping flag set incorrect in
 encoder

The cropping flag in ENC_HEADER register is bit3 on mx6, the patch
fixed it.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 vpu/vpu_util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vpu/vpu_util.c b/vpu/vpu_util.c
index 450cdee..41d4680 100644
--- a/vpu/vpu_util.c
+++ b/vpu/vpu_util.c
@@ -551,7 +551,7 @@ void EncodeHeader(EncHandle handle, EncHeaderParam * encHeaderParam)
 
 	if (cpu_is_mx6q()) {
 		VpuWriteReg(CMD_ENC_HEADER_CODE, encHeaderParam->headerType |
-			frameCroppingFlag << 2);
+			frameCroppingFlag << 3);
 	} else {
 		if (encHeaderParam->headerType == VOS_HEADER ||
 		    encHeaderParam->headerType == SPS_RBSP) {
-- 
1.8.0

